app/build.gradle

```diff
@@ -88,14 +88,6 @@ android {
         }
     }
 
-    variantFilter { variant ->
-        def flavors = variant.flavors*.name
-        // We only need a gecko debug build for now.
-        if (flavors.contains("gecko") && (variant.buildType.name != "debug" || flavors.contains("klar"))) {
-            setIgnore(true)
-        }
-    }
-
```

Comment:
Did you need to do non-debug gecko builds? I would like to keep this so that we do not build all configurations (release, beta, ..) with gecko every time on buddybuild.